aten/src/ATen/native/Activation.cpp

```diff
@@ -13,7 +13,16 @@
 
 #include <c10/util/irange.h>
 
-namespace at { namespace native {
+namespace at {
+namespace meta {
+TORCH_META_FUNC(threshold)(const Tensor& self, const Scalar& threshold, const Scalar& value) {
+  set_output(0, self.sizes(), {}, self.options().memory_format(self.suggest_memory_format()), self.names());
```

Comment:
This will twing named tensor warnings so you should do `self.has_names() ? self.names() : at::ArrayRef<Dimname>{}` instead. Sorry it's dumb and we should fix this.

cc @zou3519